### PR TITLE
Edit transaction ready

### PR DESCRIPTION
**Pull request by Marcholio**

### CODE REVIEW

File: test/databaseTests.js [hunk 1]
```diff
@@ -288,4 +288,35 @@ describe('DATABASE TESTS', function() {
...
+        stakeholder: 'Test company Oy'
+      });
+
+      db.transaction.findById(testId).then(function(transaction) {
+        try {
+          transaction.date.should.equalDate(new Date('2017-08-29'));
+          transaction.amount.should.equal(1234.2);
+          transaction.description.should.equal('Testbusiness');
+          transaction.stakeholder.should.equal('Test company Oy');
+          done(null);
```
Comment (marinp1) [inline test/databaseTests.js:313]: Usually I've tried to use done(); but doesn't really matter.

File: server/server.js [hunk 2]
```diff
@@ -62,6 +62,12 @@ dataImporter.importData(models).then(function() {
     });
   });
 
+  app.post('/api/updateTransaction', cors(), (req, res) => {
```
Comment (marinp1) [inline server/server.js:65]: What happens on fail in all of these?

Comment (Marcholio) [reply server/server.js:65]: Server crashes, world breaks, black hole emerges

Comment (marinp1) [reply server/server.js:65]: Cool, let's leave it like this then.

**Pull request merged**